### PR TITLE
Added required packages for RAIS-mobility

### PR DESCRIPTION
**Pull request by parlerBinou**
Added three packages for the [RAIS-mobility app](https://github.com/StatCan/R-dashboards/pull/30).

- circlize and shiny.i18n are downloaded as binaries from CRAN.
- Cairo requires building from the source, but it works without any modifications in the testing.

### CODE REVIEW

File: PACKAGES [hunk 1]
```diff
@@ -45,3 +45,6 @@ ggrepel
 ggthemes
 knitr
 skimr
+shiny.i18n
```
Comment (blairdrummond) [inline PACKAGES]: I think this line is a duplicate

Comment (parlerBinou) [reply PACKAGES]: Sorry, didn't notice it was already there.
Should I resubmit a PR?

Comment (blairdrummond) [reply PACKAGES]: No need! We'll squash this PR into one commit so should be no problem

Comment (parlerBinou) [reply PACKAGES]: Didn't know it automatically detects additional commits.
I removed the entry.

Thanks!

**Review by ca-scribner — APPROVED** [review]
lgtm

**Pull request merged**